CMSgov/managed-care-review
# MR-2980 Add new division specific email config

## Summary
Product would like us to use real email addresses in the help text variables on VAL since they are displayed in emails that pilot test users onboarding see. The problem is we have intentionally used environment specific email configuration up until today.

**Change in this PR (and in AWS Parameter Store):**
- Assign new config variables to each division email address in AWS
- Use the new variables in business logic (see `generateCMSReviewerEmailers`). 
- Deprecate `ratesAddresses`. We can delete entirely from AWS once this lands in prod.
- Add documentation
- 
#### Related issues
https://qmacbis.atlassian.net/browse/MR-2980

### Configuration Changes
NEW
>#### /configuration/email/dmcp
> #### /configuration/email/oact
> #### /configuration/email/dmco
DEPRECATED
> #### /configuration/email/ratesAddresses 


#### Test cases covered
New tests
- `filterChipAndPRSubmissionReviewers` and `includesChipProgram` have unit tests now
- each type of email has unit tests for `to addresses list includes all division emails for contract and rate package` as well as `to addresses list does not include help addresses`

## QA guidance
- 🔍 Check that email senders are still working as expected. Also, could someone please spot double check the oact, dmcp and dcmo config in dev/val/prod in AWS Parameter Store? A second person going through and making sure things look proper there before I merge would be nice. 
- 📓  Read documentation make sure its accurate
- 🐛 This has to merged through to fix the bug. Only then I can change the help address sender variables in VAL safely.
